connectors/byoc.py

```diff
@@ -710,7 +710,8 @@ async def pending_jobs(self, connector_ids):
                 ]
             }
         }
-        async for job in self.get_all_docs(query=query):
+        sort = [{"created_at": "asc"}]
```

Comment:
nit: "asc"/"desc" could be an enum, so we don't need to know the exact string value all the time, if we want to resue it

Reply:
Done: https://github.com/elastic/connectors-python/pull/793/commits/572276d742c9297d59f7e582d1805f66e618e8c9